pkg/common/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+package constants
```

Comment:
let's add the license header

Reply:
agreed - will have to get used to that (setup my intellij file creation template ; )